api/wasm/cpp/README.md

```diff
@@ -127,15 +127,22 @@ all: plugin.wasm
 
 ### Ownership of the resulting .wasm files
 
-The compiled files may be owned by root.  To chown them add a line in the Makefile where ID is the desired user id (e.g. the result of "id -u"):
+The compiled files may be owned by root.  To chown them add the follow lines to the Makefile and docker invocation:
 
 ```
-DOCKER_SDK=/work/sdk
+DOCKER_SDK=/sdk
 
 all: myproject.wasm
-  chown ID.ID $^
+  chown ${uid}.${gid} $^
 
-include ${DOCKER_SDK}/Makefile.base_lite
+include /sdk/Makefile.base_lite
```

Comment:
Why this was changed from `${DOCKER_SDK}` to a hardcoded value?

Reply:
Done